src/teleop_twist_joy.cpp

```diff
@@ -150,18 +164,16 @@ TeleopTwistJoy::TeleopTwistJoy(rclcpp::Node::SharedPtr & node)
   pimpl_->sent_disable_msg = false;
 }
 
+TeleopTwistJoy::~TeleopTwistJoy()
+{
+  delete pimpl_;
+}
+
 void TeleopTwistJoy::Impl::joyCallback(const sensor_msgs::msg::Joy::SharedPtr joy_msg)
 {
+  // Initializes with zeros by default.
```

Comment:
Nit: As this is the same behavior as in ROS 1 I don't think it deserves a comment

Reply:
That's only there to reduce the differences to upstream.

Reply:
oh I missed it was present upstream. All good then :+1: